packages/ember-glimmer/lib/utils/references.js

```diff
@@ -52,6 +52,27 @@ export class UpdatableReference extends RootReference {
   }
 }
 
+export class GetHelperReference {
+  constructor(args) {
+    this.positionalArgs = args.positional;
+  }
+
+  isDirty() { return true; }
+
+  value() {
+    let objRef = this.positionalArgs.at(0).value(),
+      pathRef = this.positionalArgs.at(1).value();
+
+    return objRef ? get(objRef, pathRef) : objRef;
```

Comment:
@chadhietala yes, sounds reasonable. Anyway currently we have [this](https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/tests/helpers/get_test.js#L325-L337) `ember-htmlbars` test. 
It ensures that we support `nulls`.
So my concern here is should we support `nulls` in the `ember-glimmer` or to just mark this test as `[htmlbars]`.